apps/studio/components/interfaces/Database/Schemas/SchemaTableNode.tsx

```diff
@@ -65,13 +67,24 @@ const TableNode = ({
               <Table2 strokeWidth={1} size={12} className="text-light" />
               {data.name}
             </div>
-            {data.id && !placeholder && (
-              <Button asChild type="text" className="px-0 w-[16px] h-[16px] rounded">
-                <Link href={`/project/${data.ref}/editor/${data.id}`}>
-                  <ExternalLink size={10} className="text-foreground-light" />
-                </Link>
-              </Button>
-            )}
+            <div className="flex gap-x-1 items-center">
+              <ButtonTooltip
+                type="text"
+                className="px-0 w-[16px] h-[16px] rounded"
+                icon={<InfoIcon size={5} className="text-foreground-light !w-[10px] !h-[10px]" />}
+                onClick={() => {}}
+                tooltip={{
+                  content: { side: 'top', text: data.comment ?? 'No Description Available' },
+                }}
+              />
```

Comment:
I recommend hiding the tooltip button when there are no comments for the table to reduce cognitive load.